File: fib.php
<?php
	function fib($x)
	{
		return ($x<=1) ? 1 : fib($x-1) + fib($x-2) ;
	}

	$time=microtime(true);
	echo(fib(32)."\n".((microtime(true)-$time)*1000000))."\n";
?>

File: fib.cpp
#include <iostream>	
#include <sys/time.h>

using namespace std;

int fib(int x)
{
	return (x<=1) ? 1 : fib(x-1)+fib(x-2);
}

int main()
{
	timeval start_timer;
	timeval end_timer;
	gettimeofday(&start_timer, NULL);

	cout << fib(42) << "\n";

	gettimeofday(&end_timer, NULL);

//	cout << (end_timer.tv_usec - start_timer.tv_usec) << "\n";
	return 0;
}
